# uint8[] lidar
sensor_msgs/PointCloud2 lidar_point

# BBOX
float32[] x_min
float32[] y_min
float32[] z_min
float32[] x_max
float32[] y_max
float32[] z_max
